### PR TITLE
refactor(app): thermocycler GEN2 design qa feedback

### PR DESCRIPTION
**Pull request by jerader**
closes RCORE-242 and RUXD-512

# Overview

Address design QA feedback mentioned in ticket (change TC module images, change lid status label)
Screen Shot 2022-09-30 at 2 07 18 PM

Screen Shot 2022-10-03 at 4 11 07 PM

Screen Shot 2022-10-03 at 4 11 31 PM

# Changelog

- make `iconColor` optional in `StatusLabel` in order to make the status circle optional
- make changes in `ThermocyclerModuleData` lid status label - make it so the status circle does not appear for that particular status
- add new images

# Review requests

- check both TC GEN1 and TC GEN2 module cards, the images and status label for the lid should match the designs

# Risk assessment

low

### CODE REVIEW

File: app/src/atoms/StatusLabel/index.tsx [hunk 1]
```diff
@@ -48,23 +48,25 @@ export const StatusLabel = (props: StatusLabelProps): JSX.Element | null => {
...
-          ) : null}
-        </Icon>
+        {iconColor != null ? (
+          <Icon
+            name="circle"
+            color={iconColor}
+            size={iconSize ?? '0.25rem'}
+            data-testid="status_circle"
+          >
+            {pulse != null && pulse ? (
```
Comment (shlokamin) [inline app/src/atoms/StatusLabel/index.tsx:60]: `pulse != null && pulse` is redundant right? i think you just need `pulse != null`

File: app/src/atoms/StatusLabel/index.tsx [hunk 2]
```diff
@@ -48,23 +48,25 @@ export const StatusLabel = (props: StatusLabelProps): JSX.Element | null => {
...
-              calcMode="discrete"
-              repeatCount="indefinite"
-              data-testid="pulsing_status_circle"
-            />
-          ) : null}
-        </Icon>
+        {iconColor != null ? (
+          <Icon
+            name="circle"
+            color={iconColor}
```
Comment (shlokamin) [inline app/src/atoms/StatusLabel/index.tsx]: hmm, while this totally works, I think conditionally rendering an `Icon` component only when an `iconColor` prop is provided is a little confusing. as a consumer of this component, i would think that passing in an `iconColor` prop would just change the color from its default to the specified color (similar to how the `textColor` and `iconSize` props are used in this component).

could we instead have a `showIcon` prop that is a boolean that gets defaulted to `true`, and an optional `iconColor` prop (similar to `iconSize`) that gets defaulted to something?

**Review by shlokamin — APPROVED** [review]
LGTM, left a suggestion about reworking props but i'll leave that up to you!

File: app/src/atoms/StatusLabel/index.tsx (hunk 2 above)
Comment (jerader) [reply app/src/atoms/StatusLabel/index.tsx]: This makes a lot of sense! I'll add `showIcon`

**Pull request merged**